### PR TITLE
Adds Space Ruin: Rocky Motel

### PR DESCRIPTION
**Pull request by haydenredacted**
## What Does This PR Do

This PR adds a new space ruin, the Rocky Motel. While not an over-complicated ruin, it can provide explorers and any other space farers a quiet place with a working kitchen and atmospherics system. The Rocky Motel was made to be someone's house so the loot within it is not very rare, other than some gold and credits.

## Why It's Good For The Game

More space ruins are cool and makes exploring space more interesting.

## Images of changes

StrongDMM view:
strongdmm

In game view:
in-game

## Testing

Loaded a server, teleported to the ruin and saw it.

## Changelog
:cl:
add: Added a new space ruin, the Rocky Motel
/:cl:

### CODE REVIEW

**Comment by warriorstar-orion** [comment]
@haydenredacted initial pass:

0. The unit tests are failing because you need to add your ruin back to the example config.
1. Reinforced walls are typically used to hinder access to high-value areas. I'd use regular walls for all of this.
2. Add a wall to this corner:
2024_05_26__13_39_36__paradise dme  rocky_motel dmm  - StrongDMM
3. Add walls to these corners. You could also get rid of the wall and just have the rock be the wall:
2024_05_26__13_39_39__paradise dme  rocky_motel dmm  - StrongDMM
4. Stray windoor in this wall:
2024_05_26__13_39_50__paradise dme  rocky_motel dmm  - StrongDMM
5. Use the `/obj/structure/chair/sofa/right` and `/obj/structure/chair/sofa/left` subtypes as the edges of the couch, like so:
2024_05_26__13_40_37__paradise dme  rocky_motel dmm  - StrongDMM
6. The dark tile border around this room is incomplete:
2024_05_26__13_39_57__paradise dme  rocky_motel dmm  - StrongDMM
7. I'd extend the power cable (even though it's non-functional) in through the airlock up to the first non-plating floor, like so:
2024_05_26__13_49_47__paradise dme  rocky_motel dmm  - StrongDMM
8. Use the `/area/ruin/space/powered` type for the whole ruin, including the rock. The space area is effectively fullbright and doesn't look as good, the ruin area makes it appropriately dark (except the falsewalls apparently because they're objects, not turfs, but honestly that should be fixed)
2024_05_26__13_53_42__Paradise Station 13
9. I'd probably move the solars to be exposed to space, similarly to listeningpost.dmm.

**Review by 1080pCat — APPROVED** [review]
Looks good. Only suggestion would to be to replace this signaler
image
with the old radio prefab that meta uses in that weird tiny room off by viro, 

image

pixel shifted by 10 rather than 15

**Comment by warriorstar-orion** [comment]
@haydenredacted looks great! one small suggestion:

2024_06_04__06_51_12__paradise dme  rocky_motel dmm  - StrongDMM

put an airlock here to separate the living space from the entryway with the EVA supplies. it provides a clearer separation between the two areas and will prevent things in the living area from getting sucked out into space if the external airlocks are stuck open.

**Comment by haydenredacted** [comment]
Updated render inside of StrongDMM:
image

File: _maps/map_files/RandomRuins/SpaceRuins/rocky_motel.dmm [hunk 1]
```diff
@@ -0,0 +1,1073 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"aY" = (
+/obj/structure/chair/comfy/shuttle{
+	dir = 1
+	},
+/turf/simulated/floor/mineral/titanium,
+/area/ruin/space/powered)
+"ci" = (
+/obj/machinery/door/airlock/external,
```
Comment (warriorstar-orion) [inline _maps/map_files/RandomRuins/SpaceRuins/rocky_motel.dmm:9]: Just an interior airlock here.

```suggestion
/obj/machinery/door/airlock,
```

**Review by warriorstar-orion — APPROVED** [review]
LGTM!

**Review by AffectedArc07 — APPROVED** [review]
CFG change done on live

**Pull request merged**